README.md

```diff
@@ -551,6 +549,8 @@ extend (whether to other areas of Astronomy or in other domains).
 Release History
 ---------------
 
+4.9.1: 01 August 2017 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.825839.svg)](https://doi.org/10.5281/zenodo.825839)
```

Comment:
Okay, so this wasnt' my last comment on this. So, zenodo has a "latest" version URL then that doesn't have to be changed when there's a release? Sweet.